callbacks/associations.go

```diff
@@ -340,10 +340,11 @@ func onConflictOption(stmt *gorm.Statement, s *schema.Schema, selectColumns map[
 
 func saveAssociations(db *gorm.DB, rel *schema.Relationship, values interface{}, selectColumns map[string]bool, restricted bool, defaultUpdatingColumns []string) error {
 	var (
-		selects, omits []string
-		onConflict     = onConflictOption(db.Statement, rel.FieldSchema, selectColumns, restricted, defaultUpdatingColumns)
-		refName        = rel.Name + "."
+		onConflict = onConflictOption(db.Statement, rel.FieldSchema, selectColumns, restricted, defaultUpdatingColumns)
+		refName    = rel.Name + "."
 	)
+	selects := make([]string, 0, len(selectColumns))
```

Comment:
better to keep those codes in the `var` group?

Reply:
> better to keep those codes in the `var` group?

Thanks for the review!
You're right, I will put it back.